src/ESP8266FS.java

```diff
@@ -208,6 +208,12 @@ private void createAndUpload(){
     File espota = new File(platform.getFolder()+"/tools");
     File esptool = new File(platform.getFolder()+"/tools");
     String serialPort = PreferencesData.get("serial.port");
+    String pythonCmd;
+    if(PreferencesData.get("runtime.os").contentEquals("windows"))
+      pythonCmd = "python.exe";
+    else
```

Comment:
Minor, can you please wrap these one line statements with braces? Seems to be style used by the surrounding code.

Reply:
This was cut-n-pasted from later in the original code, but it's simple enough to do a ternary and avoid the issue.